packages/ui/src/views/chatmessage/ChatMessage.js

```diff
@@ -118,7 +118,8 @@ export const ChatMessage = ({ open, chatflowid, isDialog }) => {
 
         setLoading(true)
         setMessages((prevMessages) => [...prevMessages, { message: userInput, type: 'userMessage' }])
-        addChatMessage(userInput, 'userMessage')
+        // waiting for first chatmessage uploaded, the first chatmessage id will be chatId for every components
```

Comment:
the code `await addChatMessage(userInput, 'userMessage')` looks the same before and after changes, is the comment redundant?

Reply:
Hi, maybe I should say goodmorning to you! Considering that when the chat message is sent for the first time, if the `POST /chatmessage/{id}` request and the `POST /internal-prediction/${id}` request are executed concurrently, the first message id ORM query will fail in the prediction request.

Reply:
I tried to remove `await` and make some test, as my last comment, it will trigger error from api `/internal-prediction/{id}` sometimes, because chatmessage can't be queried in database
![image](https://github.com/FlowiseAI/Flowise/assets/22190579/5be4aba2-56d6-4fde-8acf-eacd945033f2)